incubator/elasticsearch/values.yaml

```diff
@@ -108,3 +121,10 @@ data:
     enabled: false
     # minAvailable: 1
     maxUnavailable: 1
+  readinessProbe:
+    initialDelaySeconds: 5
+    periodSeconds: 10
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 90
+    periodSeconds: 10
```

Comment:
Hello @artyomboyko, you must add a new line at the end of file for pass CI lint.